### PR TITLE
Feature/add positions tokenizer test

### PR DESCRIPTION
**Pull request by ramon-astudillo**
Modifies WordTokenizer to load rules stored in local files in __init__

Adds protected rules in

    scripts/protected

copied from 
 
    unbabel_mt/mtserver/tokenizer

### CODE REVIEW

File: tests/test_tokenizer.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        sentence_str += token
+        if end != len(sentence_str):
+            sentence_str += (' '*(end - len(sentence_str)))
+    return sentence_str
+
+
+class PositionsTokenizerTest(TestCase):
+    # u"。", u"?", u"!"
+    def test_en_tokenization(self):
+        canonical_cases = ["Hi, this is is a test"]
```
Comment (josemonteiro) [inline tests/test_tokenizer.py]: Is the repeated "is" on purpose? :)

File: unbabel_text_utils/unbabel_word_tokenizer/tokenizer.py [hunk 2]
```diff
@@ -2,17 +2,27 @@
...
+)
+
 
 class WordTokenizer(object):
     '''NOTE: it is still necessary to run the Moses utf8-normalize.sh script
     to normalize the text to canonical Unicode before tokenizing a sentence
     or file.'''
     def __init__(self, language, aggressive=False, penn=False,
-                 no_escaping=False):
+                 no_escaping=False, load_nonbreak_protected=True):
```
Comment (josemonteiro) [inline unbabel_text_utils/unbabel_word_tokenizer/tokenizer.py]: Can you split this in 2 flags? `load_nobreaking_prefixes` and `load_protected_patterns` or similar? Just to make it consistent

File: tests/test_tokenizer.py (hunk 1 above)
Comment (ramon-astudillo) [reply tests/test_tokenizer.py]: No :), the test string is also not very demanding.

File: unbabel_text_utils/unbabel_word_tokenizer/tokenizer.py (hunk 2 above)
Comment (ramon-astudillo) [reply unbabel_text_utils/unbabel_word_tokenizer/tokenizer.py]: Yep

**Pull request merged**